### ISSUE TITLE
Welcome/update messages should go to stderr

### ISSUE THREAD

**Issue by james-w** [bitnami/stacksmith-base#82]
Hi,

I think that that the welcome/update messages should go to stderr, not stdout. They aren't part
of the normal output of any command, so could potentially corrupt the output stream of the
code that the user is running.

I don't think it's likely to be critical for most of the uses that our users will have, but if they
need to pipe container output to something else then they will definitely care, and could
well ditch our images as they are interfering with their code.

Thanks,

James

**Comment by james-w** [comment]
To see the problem run

    docker run --rm -i bitnami/ruby echo "a" | wc

This outputs

        7      24     223

Whent the user only output 1 character.

**Comment by prydonius** [comment]
Agreed, and this is an issue @spidaman ran into, as well. Another option could be an environment variable to disable the messages, though outputting to `stderr` does sound more practical.

**Comment by prydonius** [comment]
cc @sameersbn

**Comment by james-w** [comment]
Yeah, I think an env var to disable would be good as well, but I think that any message should go to stderr.

**Comment by sameersbn** [comment]
Redirecting the script output to `/dev/stderr` inside the container does not make any difference in the above use-case. the default docker logging driver streams all the logs on the `/dev/stdout` and the `wc` command will consume all the rendered output. 

If we want to support the above case, we have to either avoid printing any messages or add an switch that redirects all script output to `/dev/null`.

```diff
diff --git a/rootfs/opt/bitnami/stacksmith-utils.sh b/rootfs/opt/bitnami/stacksmith-utils.sh
index e9e4020..9453a6d 100644
--- a/rootfs/opt/bitnami/stacksmith-utils.sh
+++ b/rootfs/opt/bitnami/stacksmith-utils.sh
@@ -23,7 +23,7 @@ check_for_updates() {
 # Prints the welcome page for this Bitnami Docker image
 print_image_welcome_page() {
   GITHUB_PAGE=https://github.com/bitnami/bitnami-docker-${BITNAMI_APP_NAME}
-cat << EndOfMessage
+cat >/dev/stderr << EndOfMessage
 
   *** Welcome to the ${BITNAMI_APP_NAME} image ***
   *** Brought to you by Bitnami ***
````

**Comment by sameersbn** [comment]
Resolved  in #84, thanks @james-w
